spec.emu

```diff
@@ -75,5 +75,17 @@ contributors: Chengzhong Wu, Ecma International
       </emu-clause>
 
     </emu-clause>
+    <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
+    <emu-clause id="sec-properties-of-the-error-prototype-object">
+      <h1>Properties of the Error Prototype Object</h1>
+        <emu-clause id="sec-placeholder"><h1>Placeholder to ensure correct clause numbering</h1></emu-clause>
```

Comment:
you need some extra js and css to hide these from the rendered output, as well as in the TOC.

Reply:
I believe I've bootstrapped those styles and scripts properly [here](https://github.com/tc39/proposal-error-cause/blob/main/spec.emu#L12) by copy-pasting from https://github.com/tc39/proposal-private-fields-in-in 😄

Reply:
ah, nvm :-)